app/javascript/react/components/NewMovieForm.js

```diff
@@ -0,0 +1,79 @@
+import React, {useState} from "react"
+import _ from "lodash"
+import NewMovieFormTile from "./NewMovieFormTile"
+
+const NewMovieForm = (props) => {
+  const clearForm = {
+    title: "",
+    year: "",
+    director: "",
+    image: "",
+    description: ""
+  }
+
+  const [movieRecord, setMovieRecord] = useState(clearForm)
+  const [errors, setErrors] = useState({})
+
+  const addMovie = async (formPayload) => {
+    try {
+      const response = await fetch("/api/v1/movies", {
+        credentials: "same-origin",
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+          "Accept": "application/json"
+        },
+        body: JSON.stringify(formPayload)
+      })
+      if(!response.ok){
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw(error)
+      }
+      setMovieRecord(clearForm)
+    }catch(error) {
+      console.log(`Error in fetch: ${error.message}`)
+    }
+  }
+
+  const submitHandler = async (event) => {
+    event.preventDefault()
+    if (validForSubmission()) {
+      addMovie(movieRecord)
```

Comment:
what happens after the movie submits? are we left on the new movie form page with a blank form? thinking about the user experience we should redirect to a page where they can see the new info, like to the index or the show page for the new movie